From 791e12d2b3c47fdc7345d5dbdef98764a424dcd8 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Thu, 13 Sep 2018 13:20:05 +0800
Subject: [PATCH 4604/5242] MLK-19566 pwm: imx: Add NXP copyright

commit  3256046b78b20037488b9d3f0010528ae85fdda3 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds NXP copyright for the i.MX PWM driver.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
(cherry picked from commit 59c94789cef003fe64ebeb68c7b209da0bf2de78)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/pwm/pwm-imx.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/pwm/pwm-imx.c b/drivers/pwm/pwm-imx.c
index 7bbc9d3..1aadaf1 100644
--- a/drivers/pwm/pwm-imx.c
+++ b/drivers/pwm/pwm-imx.c
@@ -1,4 +1,6 @@
 /*
+ * Copyright 2017-2018 NXP
+ *
  * simple driver for PWM (Pulse Width Modulator) controller
  *
  * This program is free software; you can redistribute it and/or modify
-- 
1.7.9.5

